Controllers/Error.Controller.js

```diff
@@ -0,0 +1,13 @@
+function erroHandler(err, req, res, next){
+    console.log(err.stack);
+
+    if(err.name === 'NotFoundError'){
+        return res.status(404).json({ msg: "Not Found" });
+    }else if (err.name === "BadRequestError") {
+        return res.status(400).json({ msg: "Bad Request" });
+    }
```

Comment:
Please handle bad requests like this using the SQL error code (e.g. P2202). 
We would like you to be using middleware functions for error handling.